File: python/Course.py
class Course:
    # deklarasi property private
    __nama_matkul = ""

    # membuat konstruktor
    def __init__(self, namamatkul = ""):
        self.__nama_matkul = namamatkul
    
    # membuat getter
    def getNamamatkul(self):
        return self.__nama_matkul

File: c++/Course.cpp
#include <string>
using namespace std;

#ifndef COURSE_CPP_H
#define COURSE_CPP_H

class Course{
    private:
        string namamatkul;
    
    public:
        Course(){

        }

        Course(string namamatkul){
            this->namamatkul = namamatkul;
        }

        string getNamaMatkul(){
            return this->namamatkul;
        }

        void setNamaMatkul(string namamatkul){
            this->namamatkul = namamatkul;
        }

        ~Course(){
            
        }
};
#endif